Nimble/Utils/NotificationCollector.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  NotificationCollector.swift
+//  Nimble
+//
+//  Created by Brian Gerstle on 12/31/15.
+//  Copyright © 2015 Jeff Hui. All rights reserved.
+//
```

Comment:
Nitpick: Please remove the header comments. Mostly just to be consistent with the rest of the code base (and they don't add much value)